### DISCUSSION TITLE
Is there a way to print thin inner walls instead of creating gaps?

### DISCUSSION THREAD

**Discussion by d-schmidt** [smartavionics/Cura#120] [Q&A] [answered]
Compensations does only work when two outer or two inner walls meet. It does nothing when the outer wall takes space from the inner wall

The yellow parts should be green:
![grafik](https://user-images.githubusercontent.com/3226483/129471407-ff911086-d136-4267-8c48-d4117d28b6e2.png)

**Comment by d-schmidt** [comment]
I assume I have to wait for Arachne?

**Comment by Sophist-UK** [comment, marked as answer]
Not sure what your nozzle size is or what your line sizes are for outer and inner walls, but...

You can try reducing the outer wall size by a smidgen (say 0.01mm) and see if that helps. Or do the same for inner wall.

**Reply by d-schmidt** [reply]
I had that same idea after a while. I lowered the inner wall line width a tiny bit and upped the inner wall flow to compensate.
